components/error-view.js

```diff
@@ -27,14 +31,16 @@ module.exports = React.createClass({
       time: 0,
       tabId: this.props.tabId
     });
-  },
-  componentWillMount: function() {
+  }
+
+  componentWillMount() {
     sendMetricsEvent('error_view', 'render');
-  },
-  render: function() {
+  }
+
+  render() {
     return (
-        <div className='error' onMouseEnter={this.enterView} onMouseLeave={this.leaveView}>
-          <div className={cn('controls', 'drag', {hidden: !this.state.hovered, minimized: this.props.minimized})}>
+        <div className='error' onMouseEnter={this.enterView.bind(this)} onMouseLeave={this.leaveView.bind(this)}>
```

Comment:
Not when using es6 classes, https://facebook.github.io/react/docs/handling-events.html